remote_repository_test.go

```diff
@@ -23,10 +23,6 @@ func TestNewRemoteRepository(t *testing.T) {
 		valid:      true,
 		vcsBackend: GitBackend,
 		repoURL:    "https://github.com/motemen/ghq",
-	}, {
```

Comment:
Do not delete this test. We need it for self-hosted VCS environments. You can change the test case as appropriate.